cmd/build/helmify/static/README.md

```diff
@@ -144,9 +144,11 @@ information._
 | validatingWebhookObjectSelector                            | The label selector to further refine which namespaced resources will be selected by the webhook. Please note that an exemption label means users can circumvent Gatekeeper's validation webhook unless measures are taken to control how exemption labels can be set.                          | `{}`                                                                                                                                                                  |
 | validatingWebhookMatchConditions                           | The match conditions written in CEL to further refine which resources will be selected by the webhook. All match conditions must evaluate to true for the webhook to be called                                                                                                                 | `[]`                                                                                                                                                                  |
 | validatingWebhookCheckIgnoreFailurePolicy                  | The failurePolicy for the check-ignore-label validating webhook                                                                                                                                                                                                                                | `Fail`                                                                                                                                                                |
+| validatingWebhookCheckIgnoreScope                          | The scope for the check-ignore-label validating webhook                                                                                                                                                                                                                                        | `*`                                                                                                                                                                   |
```

Comment:
Appologies on going back and forth on this, but allowing users to set the scope for this might nont be a good idea. Since the scope of the check-ignore-label validating webhook should not be set to `Namespaced`, as this scope applies only to rules for Namespace resources. Setting it to `Namespaced` will prevent the rejection of invalid Namespaces with the `admission.gatekeeper.sh/ignore` label. Namespaces with this label, along with all resources within them, are exempt from Gatekeeper validation. We should revert this back to remove the variable.

Reply:
I removed the variable. I left `HELMSUBST_VALIDATING_WEBHOOK_CHECK_IGNORE_OPERATION_RULES` in the code since it looks clearer to me that way - but the scope can only be `*` (default value in K8s).